packages/expo-module-scripts/package.json

```diff
@@ -7,7 +7,7 @@
     "expo-module": "bin/expo-module.js"
   },
   "scripts": {
-    "lint": "expo-module lint ."
```

Comment:
What is the reasoning behind this change?

Reply:
`expo-module lint .` actually executes [`eslint src`](https://github.com/expo/expo/blob/381574231803220af631c0133741d7c3e8062a60/packages/expo-module-scripts/bin/expo-module-lint#L20-L21), and [`eslint plugin`](https://github.com/expo/expo/blob/381574231803220af631c0133741d7c3e8062a60/packages/expo-module-scripts/bin/expo-module-lint#L10-L13), if it exists.

Unfortunately, this package doesn't have either. So ESLint actually warned us that no files could be found. This changes the execution to `eslint .`, to actually lint all files within this project.